packages/SwingSet/src/kernel/liveSlots.js

```diff
@@ -1220,9 +1231,14 @@ function build(
         retireImports(vrefs);
         break;
       }
+      case 'startVat': {
+        result = startVat();
+        break;
+      }
       default:
         assert.fail(X`unknown delivery type ${type}`);
     }
+    return result;
```

Comment:
We need to update the type comment for `dispatchToUserspace`, I think this new case makes it `{ void | Promise<void> }`. We don't have `// @ts-check` turned on here yet, else I bet it would have complained.